docs/public/_redirects

```diff
@@ -441,3 +442,5 @@ https://v4.material-ui.com/* https://v4.mui.com/:splat 301!
 
 ### MUI Design kits
 /figma/getting-started/ https://mui-org.notion.site/MUI-for-Figma-docs-4eb22360388047a9a31fc19d300a285e 302
+
+/* https://mui.com/:splat 301!
```

Comment:
Should we use a domain name here? See https://github.com/mui/material-ui/pull/36296#discussion_r1116279746

```suggestion
https://v5.mui.com/* https://mui.com/:splat 301!
```